src/kmedoids.jl

```diff
@@ -23,13 +23,28 @@ The output of [`kmedoids`](@ref) function.
 mutable struct KmedoidsResult{T} <: ClusteringResult
     medoids::Vector{Int}        # indices of methods (k)
     assignments::Vector{Int}    # assignments (n)
-    acosts::Vector{T}           # costs of the resultant assignments (n)
+    costs::Vector{T}           # costs of the resultant assignments (n)
     counts::Vector{Int}         # number of points assigned to each cluster (k)
     totalcost::Float64          # total assignment cost (i.e. objective) (k)
     iterations::Int             # number of elapsed iterations
     converged::Bool             # whether the procedure converged
 end
 
+# FIXME remove after deprecation period for acosts
+Base.propertynames(kmed::KmedoidsResult, private::Bool = false) =
+    (:medoids, :assignments, :costs, :counts, :totalcost, :iterations, :converged,
+     #= deprecated =# :acosts)
```

Comment:
I think it was not available when I was writing *Hclust* deprecations, but now one can write
```suggestion
    (fieldnames(kmed)..., #= deprecated =# :acosts)
```
which would allow modifying `KmedoidsResult` without touching this method